certified-connectors/Baron Blue/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1492 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Baron Blue",
+    "description": "Baron Blue allows you to perform complex operations in only one action. Convert JSON, XML, CSV, XLS, XLSX, and YAML between themselves in any combination. Convert to and from Excel Dates, find information in strings, find content-types and extensions, convert IP addresses (range, CIDR), and much more. New actions come monthly, and if you\u2019re a part of the manueltgomes.com/club, you\u2019ll get the connector for free, so be sure to check it out.",
```

Comment:
@mtgomes unicode symbols are not allowed. Please look for `\u` symbols and replace them. The description itself should be a bit more concise and reflect the main value and purpose offered by the connector.

Reply:
@mtgomes please fix the description and remove the "\u" symbols.